diff --git a/node_modules/@zondax/zemu/dist/emulator.js b/node_modules/@zondax/zemu/dist/emulator.js
index 457127e..07256ed 100644
--- a/node_modules/@zondax/zemu/dist/emulator.js
+++ b/node_modules/@zondax/zemu/dist/emulator.js
@@ -205,8 +205,10 @@ var EmuContainer = /** @class */ (function () {
                                 AttachStderr: true,
                                 User: '1000',
                                 Env: environment,
-                                PortBindings: portBindings,
-                                Binds: dirBindings,
+                                HostConfig: {
+                                      PortBindings: portBindings,
+                                      Binds: dirBindings,
+                                },
                                 Cmd: [command]
                             })];
                     case 1:
