diff --git a/node_modules/@ledgerhq/cryptoassets/data/dapps/ethereum.js b/node_modules/@ledgerhq/cryptoassets/data/dapps/ethereum.js
index 540ee79..5c092cc 100644
--- a/node_modules/@ledgerhq/cryptoassets/data/dapps/ethereum.js
+++ b/node_modules/@ledgerhq/cryptoassets/data/dapps/ethereum.js
@@ -12721,6 +12721,396 @@ module.exports = {
       }
     ]
   },
+  "0x11111112542d85b3ef69ae05771c2dccff4faa26": {
+    // Unoswap
+    "0x2e95b6c8": {
+      "plugin": "1inch",
+      "serialized_data": "0531696e636811111112542d85b3ef69ae05771c2dccff4faa262e95b6c8",
+      "signature": "30450221009a0afd48e31d3e0ec2d7c4c362b293f8893dbcacb5b705ecb92370a031bcd85e0220697a15b3eb546033a167a476c9c7cd8c24359585e6316563dadfbacaf6e3b520",
+      "erc20OfInterest": [
+          "srcToken"
+      ]
+    },
+    "0x7c025200": {
+      // Swap
+      "plugin": "1inch",
+      "serialized_data": "0531696e636811111112542d85b3ef69ae05771c2dccff4faa267c025200",
+      "signature": "30450221009a0afd48e31d3e0ec2d7c4c362b293f8893dbcacb5b705ecb92370a031bcd85e0220697a15b3eb546033a167a476c9c7cd8c24359585e6316563dadfbacaf6e3b520",
+      "erc20OfInterest": [
+         "desc.srcToken",
+         "desc.dstToken"
+      ]
+    },
+    "abi": [
+      {
+        "anonymous": false,
+        "inputs": [
+          {
+            "indexed": false,
+            "internalType": "string",
+            "name": "reason",
+            "type": "string"
+          }
+        ],
+        "name": "Error",
+        "type": "event"
+      },
+      {
+        "anonymous": false,
+        "inputs": [
+          {
+            "indexed": true,
+            "internalType": "address",
+            "name": "previousOwner",
+            "type": "address"
+          },
+          {
+            "indexed": true,
+            "internalType": "address",
+            "name": "newOwner",
+            "type": "address"
+          }
+        ],
+        "name": "OwnershipTransferred",
+        "type": "event"
+      },
+      {
+        "anonymous": false,
+        "inputs": [
+          {
+            "indexed": false,
+            "internalType": "address",
+            "name": "sender",
+            "type": "address"
+          },
+          {
+            "indexed": false,
+            "internalType": "contract IERC20",
+            "name": "srcToken",
+            "type": "address"
+          },
+          {
+            "indexed": false,
+            "internalType": "contract IERC20",
+            "name": "dstToken",
+            "type": "address"
+          },
+          {
+            "indexed": false,
+            "internalType": "address",
+            "name": "dstReceiver",
+            "type": "address"
+          },
+          {
+            "indexed": false,
+            "internalType": "uint256",
+            "name": "spentAmount",
+            "type": "uint256"
+          },
+          {
+            "indexed": false,
+            "internalType": "uint256",
+            "name": "returnAmount",
+            "type": "uint256"
+          }
+        ],
+        "name": "Swapped",
+        "type": "event"
+      },
+      {
+        "inputs": [],
+        "name": "destroy",
+        "outputs": [],
+        "stateMutability": "nonpayable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "contract IAggregationExecutor",
+            "name": "caller",
+            "type": "address"
+          },
+          {
+            "components": [
+              {
+                "internalType": "contract IERC20",
+                "name": "srcToken",
+                "type": "address"
+              },
+              {
+                "internalType": "contract IERC20",
+                "name": "dstToken",
+                "type": "address"
+              },
+              {
+                "internalType": "address",
+                "name": "srcReceiver",
+                "type": "address"
+              },
+              {
+                "internalType": "address",
+                "name": "dstReceiver",
+                "type": "address"
+              },
+              {
+                "internalType": "uint256",
+                "name": "amount",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "minReturnAmount",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "flags",
+                "type": "uint256"
+              },
+              {
+                "internalType": "bytes",
+                "name": "permit",
+                "type": "bytes"
+              }
+            ],
+            "internalType": "struct AggregationRouterV3.SwapDescription",
+            "name": "desc",
+            "type": "tuple"
+          },
+          {
+            "internalType": "bytes",
+            "name": "data",
+            "type": "bytes"
+          }
+        ],
+        "name": "discountedSwap",
+        "outputs": [
+          {
+            "internalType": "uint256",
+            "name": "returnAmount",
+            "type": "uint256"
+          },
+          {
+            "internalType": "uint256",
+            "name": "gasLeft",
+            "type": "uint256"
+          },
+          {
+            "internalType": "uint256",
+            "name": "chiSpent",
+            "type": "uint256"
+          }
+        ],
+        "stateMutability": "payable",
+        "type": "function"
+      },
+      {
+        "inputs": [],
+        "name": "owner",
+        "outputs": [
+          {
+            "internalType": "address",
+            "name": "",
+            "type": "address"
+          }
+        ],
+        "stateMutability": "view",
+        "type": "function"
+      },
+      {
+        "inputs": [],
+        "name": "renounceOwnership",
+        "outputs": [],
+        "stateMutability": "nonpayable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "contract IERC20",
+            "name": "token",
+            "type": "address"
+          },
+          {
+            "internalType": "uint256",
+            "name": "amount",
+            "type": "uint256"
+          }
+        ],
+        "name": "rescueFunds",
+        "outputs": [],
+        "stateMutability": "nonpayable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "contract IAggregationExecutor",
+            "name": "caller",
+            "type": "address"
+          },
+          {
+            "components": [
+              {
+                "internalType": "contract IERC20",
+                "name": "srcToken",
+                "type": "address"
+              },
+              {
+                "internalType": "contract IERC20",
+                "name": "dstToken",
+                "type": "address"
+              },
+              {
+                "internalType": "address",
+                "name": "srcReceiver",
+                "type": "address"
+              },
+              {
+                "internalType": "address",
+                "name": "dstReceiver",
+                "type": "address"
+              },
+              {
+                "internalType": "uint256",
+                "name": "amount",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "minReturnAmount",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "flags",
+                "type": "uint256"
+              },
+              {
+                "internalType": "bytes",
+                "name": "permit",
+                "type": "bytes"
+              }
+            ],
+            "internalType": "struct AggregationRouterV3.SwapDescription",
+            "name": "desc",
+            "type": "tuple"
+          },
+          {
+            "internalType": "bytes",
+            "name": "data",
+            "type": "bytes"
+          }
+        ],
+        "name": "swap",
+        "outputs": [
+          {
+            "internalType": "uint256",
+            "name": "returnAmount",
+            "type": "uint256"
+          },
+          {
+            "internalType": "uint256",
+            "name": "gasLeft",
+            "type": "uint256"
+          }
+        ],
+        "stateMutability": "payable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "newOwner",
+            "type": "address"
+          }
+        ],
+        "name": "transferOwnership",
+        "outputs": [],
+        "stateMutability": "nonpayable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "contract IERC20",
+            "name": "srcToken",
+            "type": "address"
+          },
+          {
+            "internalType": "uint256",
+            "name": "amount",
+            "type": "uint256"
+          },
+          {
+            "internalType": "uint256",
+            "name": "minReturn",
+            "type": "uint256"
+          },
+          {
+            "internalType": "bytes32[]",
+            "name": "",
+            "type": "bytes32[]"
+          }
+        ],
+        "name": "unoswap",
+        "outputs": [
+          {
+            "internalType": "uint256",
+            "name": "returnAmount",
+            "type": "uint256"
+          }
+        ],
+        "stateMutability": "payable",
+        "type": "function"
+      },
+      {
+        "inputs": [
+          {
+            "internalType": "contract IERC20",
+            "name": "srcToken",
+            "type": "address"
+          },
+          {
+            "internalType": "uint256",
+            "name": "amount",
+            "type": "uint256"
+          },
+          {
+            "internalType": "uint256",
+            "name": "minReturn",
+            "type": "uint256"
+          },
+          {
+            "internalType": "bytes32[]",
+            "name": "pools",
+            "type": "bytes32[]"
+          },
+          {
+            "internalType": "bytes",
+            "name": "permit",
+            "type": "bytes"
+          }
+        ],
+        "name": "unoswapWithPermit",
+        "outputs": [
+          {
+            "internalType": "uint256",
+            "name": "returnAmount",
+            "type": "uint256"
+          }
+        ],
+        "stateMutability": "payable",
+        "type": "function"
+      },
+      {
+        "stateMutability": "payable",
+        "type": "receive"
+      }
+    ]
+  },
   "0x1bd435f3c054b6e901b7b108a0ab7617c808677b": {
     "0x8f00eccb": {
       "plugin": "Paraswap",
